packages/design-system/src/styles/components/_Button.scss

```diff
@@ -13,40 +13,46 @@
   font-weight: var(--fontWeight, $button__font-weight);
   line-height: 1.3;
   text-align: center;
-  text-decoration-color: transparent;
   display: inline-block;
   padding: var(--space, $spacer-1 $spacer-3);
 
+  &:is(a) {
```

Comment:
This selector is [supported in all modern browsers](https://caniuse.com/css-matches-pseudo).